src/site/stages/build/drupal/graphql/vetCenter.graphql.js

```diff
@@ -0,0 +1,83 @@
+const vetCenterFragment = `
+ fragment vetCenterFragment on NodeVetCenter {
+        entityId
+        entityUrl {
+          path
+          routed
+        }
```

Comment:
There is a shared list of properties that we generally drop into GraphQL queries that represent entire pages, `entityElementsForPages.graphql`. Might be worth checking out